### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_Compare Details_Keyboard: Scroll bar is not accessible using keybaord under the "Compare details".

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#6801]
“Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”
Environment Details:
Application Name: Azure Data Studio
Application Version: 1.9.0 (user setup)
Commit: 78a42e1
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.18362
Additional Details : Keyboard, MAS2.1.1

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Schema Compare extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Schema compare from Context menu.
8. Navigate till Select Source or Select Target & hit enter.
9. Provide all the details for Source & Target.
10. Navigate till Ok button & hit enter.
11. Navigate till Compare menu from Schema compare Menu bar & hit enter.
12. Navigate inside comparison details & select any row.
13. Navigate to "Compare details window" and using up/down arrow keys try to navigate to the editor of the "Compare details window".

**Actual:**
When we are navigating to "Compare details window" editor using up/down arrow keys after the 7th line keyboard focus is not visible and scroll bar is also not accessible by keyboard.

**Expected:**
When we are navigating to "Compare details window" editor using up/down arrow keys after the 7th line keyboard focus should be visible and scroll bar should be accessible by keyboard.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will not be able to know the complete information of the editor window.

**MAS Reference:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}](url)

**Attachment for Reference:**

**Comment by fsteffi** [comment]
Using up/down arrow keys able to access scroll bar hence closing this issue.
